src/spikeinterface/curation/auto_merge.py

```diff
@@ -572,6 +678,138 @@ def get_potential_auto_merge(
     )
 
 
+def auto_merge_units(
+    sorting_analyzer: SortingAnalyzer,
+    presets: list | None = ["similarity_correlograms"],
+    steps_params: dict = None,
+    steps: list[str] | None = None,
+    apply_merge_kwargs: dict = {},
+    recursive: bool = False,
+    merging_strategy: str = "warning",
+    extra_outputs: bool = False,
+    force_copy: bool = True,
+    **job_kwargs,
+) -> SortingAnalyzer:
+    """
+    Wrapper to conveniently be able to launch several presets for auto_merge_units in a row, as a list.
```

Comment:
```suggestion
    Automatically finds and apply merges.
    This function enables one to launch several merging presets in sequence and also to apply each
    step recursively.
```
I think we shouls make this docstring more high level since this will be the main merging entry point :)